### PR TITLE
Feature challenge 02

### PR DESCRIPTION
**Pull request by rafiahmadjan**
Created Ansible playbook to install and start Nginx web server on the local host

### CODE REVIEW

File: challenge-02/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Ansible Playbook which automatically installs and runs: Nginx web server.
+
+How to run it: ansible-playbook nginx_install_run.yml
```
Comment (zeusbaba) [inline challenge-02/README.md:3]: minor: you can use markdown for the code snippet.  
e.g.  
`ansible-playbook nginx_install_run.yml`

File: challenge-02/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+*Note:* This Playbook require root permission and only run on localhost
+
+*How to veriy nginx is running*: service nginx status
```
Comment (zeusbaba) [inline challenge-02/README.md:7]: minor: code snippets require markdown

**Review by zeusbaba — APPROVED** [review]
very good job, well done

File: challenge-02/README.md (hunk 1 above)
Comment (rafiahmadjan) [reply challenge-02/README.md:3]: Thanks for the tip

File: challenge-02/README.md (hunk 2 above)
Comment (rafiahmadjan) [reply challenge-02/README.md:7]: Thanks for the tip

**Comment by zeusbaba** [comment]
it looks good, feel free to merge it.  
tip: `squash & merge` is preferred

**Pull request merged**